.gitmodules

```diff
@@ -17,3 +14,7 @@
 	path = AI/Skirmish/BARb
 	url = https://github.com/rlcevg/CircuitAI.git
 	branch = barbarian
+[submodule "AI/Skirmish/CircuitAI"]
+	path = AI/Skirmish/CircuitAI
+	url = https://github.com/rlcevg/CircuitAI.git
```

Comment:
please revert this change. we had it several times that required submodules disappeared because of renames or deletion. thats why all submodules are hosted at https://github.com/spring/*

Reply:
Talk to @rlcevg (lamer). He is the author of both AIs and this PR, I suppose he knows better.

P.S. @abma check your springrts PMs, there is an emergency waiting your attention for a while now.

P.S.S I'm on PTO, but once I'm back I'm gonna start putting new commits to this branch as the review has stalled.

Reply:
How the heck BARbarIAn and CircuitAI that will use new or changed API functions (but not yet) ended up in spring repo?
Is Beyond-All-Reason/spring/BAR going to be merged completely into spring/spring/transition? (if so then reverting is fine)

Reply:
Ok, it is not outside of my control that this is taking so long. Had I knew this beforehand, I'd have made this branch separate from BAR live one.

Let's agree on the following: once the lamer's changes to AI have made it to the spring/spring (as part of this branch), we will certainly revert submodules URL back.

Reply:
Alternatively once I'm back we can craft a separate spring/spring AI PR and I can quickly merge it to BAR too. This way submodules change won't be needed.

Reply:
Alternatively once I'm back we can craft a separate spring/spring AI PR and I can quickly merge it to BAR too. This way submodules change won't be needed.

Reply:
dependency for submodule revert: https://github.com/spring/CircuitAI/pull/19